js/common/tests/shell/shell-view-arangosearch-link-noncluster.js

```diff
@@ -118,15 +115,12 @@ function IResearchLinkSuite () {
 
       try {
         var view = db._createView("badView", "arangosearch", meta);
+        fail();
       } catch(err) {
         assertEqual(ERRORS.ERROR_BAD_PARAMETER.code, err.errorNum);
```

Comment:
I thought a view was still supposed to be created even if some links failed creation?